logging/logging_unexported_test.go

```diff
@@ -302,7 +302,7 @@ func TestMonitoredResource(t *testing.T) {
 		{
 			"projects/P",
 			&mrpb.MonitoredResource{
-				Type:   "project",
+				Type:   "global",
```

Comment:
Is this going to be a breaking change for customers, seems like a behaviour changes.

Reply:
Sending `"project"` as a resource type is just invalid. This function generates a global resource type structure which has to have resource type to be set to "global". For folder, organization and billing account I left the resource type "as is" because it was originally wrong to expose this functionality to end user. And although these resource types are invalid, they can remain invalid because they have not be used in the first place.
However, if a user declare the properly formatted project's parent for a global resource, it should be reflected as "global" resource type and not as "project".